cpp/src/table/row_operators.cu

```diff
@@ -194,6 +210,19 @@ auto decompose_structs(table_view table,
 
       for (auto const& branch : flattened) {
         column_view temp_col = *branch.back();
+
+        // Change `Struct<List,...>` into empty struct type `Struct<>`.
+        if (temp_col.type().id() == type_id::STRUCT &&
+            (temp_col.num_children() > 0 && temp_col.child(0).type().id() == type_id::LIST)) {
+          temp_col = column_view(temp_col.type(),
+                                 temp_col.size(),
+                                 temp_col.head(),
+                                 temp_col.null_mask(),
+                                 UNKNOWN_NULL_COUNT,
```

Comment:
Can the null count be known here?

Reply:
It can be queried by ` temp_col.null_count()` which may trigger a kernel launch (now). Good point, I think with our plan to remove `UNKNOWN_NULL_COUNT` then using ` temp_col.null_count()` here would avoid modifying it again in the future.